Index: 04/src/olsen/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- 04/src/olsen/Main.java	(revision 81a57ba9e6d89f11ecab4f0f67eb39732254b64c)
+++ 04/src/olsen/Main.java	(date 1589253179357)
@@ -4,6 +4,6 @@
 
     public static void main(String[] args) {
         System.out.println("Hello, my name is Briana");
-
+        System.out.println("Testing out pushing on intellij");
     }
 }
